repos/spack_repo/builtin/packages/libmesh/package.py

```diff
@@ -224,8 +226,8 @@ def configure_args(self):
             options.append("--enable-netcdf=no")
 
         if self.spec.satisfies("+vtk"):
-            options.append("--enable-vtk")
-            options.append("--with-vtk=%s" % self.spec["vtk"].prefix)
+            options.append("--enable-vtk-required")
+            options.append("VTK_DIR=%s" % self.spec["vtk"].prefix)
```

Comment:
@bernhardkaindl Are we enforcing f-strings?

Reply:
Not that I heard of. In this case, keeping the % is pretty harmless for me (either way).
Just `.format(...)` is really distracting and should be avoided.

> Did you test with vtk~mpi when ~mpi? It might only be needed when +vtk+mpi.

Thanks for this comment, @capitalaslash: Please address that other comment, thanks!